src/cap32.cpp

```diff
@@ -116,6 +116,11 @@ byte *membank_config[8][4];
 FILE *pfileObject;
 FILE *pfoPrinter;
 
+// Store shifted keys for correct remapping where shift is released before key release. 
+// Indexed by SDL_Scancode: https://wiki.libsdl.org/SDLScancodeLookup
+static constexpr int kSDL_Scancode_Max = 284;
```

Comment:
I don't think you really want to use scancode. We don't use it anywhere.
I think you need the SDL_Key.
The reason is that they are bound to a key position to the keyboard (e.g where the Q is on a QWERTY keyboard, which would be a A on an AZERTY) whereas we adapt to the keyboard layout used by the user.

When using the SDL_Key, you can use SDLK_LAST for that:
https://www.libsdl.org/release/SDL-1.2.15/include/SDL_keysym.h
(but that's one more thing that will be complicated to migrate to SDL 2: https://wiki.libsdl.org/MigrationGuide#Input)